src/lib/components/forms/form/form-components/double-input/double-input.component.spec.ts

```diff
@@ -1,24 +1,162 @@
+    });
+
+    it('should disable the input when #isDisabled is set', async () => {
+        expect(await input.isDisabled()).toBe(false);
+        component.isDisabled = true;
+        fixture.detectChanges();
+
+        expect(await input.isDisabled()).toBe(true);
+    });
+
+    it('should have #name as label of the input', async () => {
+        const formField: MatFormFieldHarness = await loader.getHarness(MatFormFieldHarness);
+        expect(await formField.getLabel()).toBe('');
+
+        component.name = 'My Label';
+        fixture.detectChanges();
+
+        expect(await formField.getLabel()).toBe(component.name);
+    });
+
+    describe('with tooltip', () => {
+        let tooltip: MockTooltipDirective;
+        beforeEach(() => {
+            tooltip = fixture.debugElement.query(By.directive(MockTooltipDirective)).injector.get(MockTooltipDirective);
+        });
+
+        it('should apply the tcTooltip directive to the form field with the given #tooltipText', () => {
+            expect(tooltip).toBeTruthy();
+            expect(tooltip.tcTooltip).toBe('');
+
+            component.tooltipText = 'My tooltip';
+            fixture.detectChanges();
+
+            expect(tooltip.tcTooltip).toBe(component.tooltipText);
+        });
+
+        it(`should set the tooltip's placement according to the input #tooltipPlacement`, () => {
+            expect(tooltip.placement).toBe(TerraPlacementEnum.TOP);
+
+            component.tooltipPlacement = TerraPlacementEnum.BOTTOM;
+            fixture.detectChanges();
+
+            expect(tooltip.placement).toBe(component.tooltipPlacement);
+        });
+    });
+
+    it('should update the value of the input when writing a new value via `writeValue()`', async () => {
+        const value: number = 2.0;
+        component.writeValue(value);
+        fixture.detectChanges();
+
+        expect(await input.getValue()).toEqual(value.toString());
+    });
+
+    it('should call registered change callback whenever the value of the input is changed by the user', async () => {
+        const onChangeCallback: jasmine.Spy = jasmine.createSpy('onChange');
+        component.registerOnChange(onChangeCallback);
+
+        const value: number = 2.0;
+        await input.setValue(value.toString());
+
+        expect(onChangeCallback).toHaveBeenCalledWith(value);
+    });
+
+    it('should call registered touched callback whenever the input was blurred', async () => {
+        const onTouchedCallback: jasmine.Spy = jasmine.createSpy('onTouched');
+        component.registerOnTouched(onTouchedCallback);
+
+        await input.blur();
+
+        expect(onTouchedCallback).toHaveBeenCalled();
+    });
+
+    it('should be invalid when #decimalCount is disregarded', async () => {
+        const formField: MatFormFieldHarness = await loader.getHarness(MatFormFieldHarness);
+        expect(await formField.isControlValid()).toBe(true);
+        component.writeValue(2.3422);
```

Comment:
könnte sein, dass du hier stattdessen
```suggestion
        await input.setValue(2.3422);
```
machen müsstest.

Reply:
ich denke, wir brauchen es nicht, oder?

Reply:
Es gibt halt einen Unterschied. `input.setValue` gaukelt vor dass der Nutzer etwas eingibt. `component.writeValue` stattdessen würde vom Programm genutzt werden um den Wert des Inputs zu verändern.
Aber grundsätzlich sollte das control invalid sein, wenn ein falscher Wert eingetragen wird (egal welche der beiden Schnittstellen genutzt wird). 
Entsprechend kannst du es so lassen, ja :)